admin_manual/configuration/server/security/password_policy.rst

```diff
@@ -0,0 +1,61 @@
+===============
+Password Policy
+===============
+
+.. _password_policy_label:
+
+.. figure:: ../../../images/configuration/server/security/password-policy-app.png
+   :alt: The Password Policy application
+
+.. caution:: The Security and Password Policy apps are mutually exclusive.
+
+From the 2.0.0 release of `the Password Policy app`_, all ownCloud users (both enterprise and community) have the option of installing and enabling the application.
```

Comment:
Was there a reason not to use:
``all ownCloud users (both enterprise and community) have the option`` -->
``both enterprise and community editions have the option``

It is dangerous to write ``users`` even true. This makes admins nervous ! 😄

Reply:
only admins can install, not users...